GitCommands/Settings/AppSettings.cs

```diff
@@ -1825,7 +1825,18 @@ public static DiffListSortType DiffListSorting
 
         public static string GetGitExtensionsFullPath()
         {
-            Debug.Assert(_applicationExecutablePath.EndsWith("GitExtensions.exe"), $"{_applicationExecutablePath} must point to GitExtensions.exe");
+#if DEBUG
+#pragma warning disable SA1515 // Single-line comment should be preceded by blank line
```

Comment:
Follow the rules even if they are annoying here...
Otherwise the workaround is fine for me